tasks/install.yum.yml

```
@@ -1,3 +1,28 @@
 ---
 
-- fail: msg="RedHat is not supported now"
+- name: Install EPEL repository
+  yum:
+    name: epel-release
+  when: openvpn_use_external_repo
+
+- name: Install requirements (RedHat)
+  apt: name={{item}} force=yes
+  with_items: [openvpn, openssl, zip]
+
+- name: Install dependencies (RedHat)
+  apt: name={{item}} force=yes
+  when: openvpn_use_pam_users|default(false)
```

I wonder,

```suggestion
  when: openvpn_use_pam_users|length
```

won't do the same thing? I know this is the way it is in `install.deb.yml` too, but let's discuss overall improvements.

---

You're right... maybe i overseen that openvpn_use_pam_users is an list, not an dict.

---

You're right... maybe i overseen that openvpn_use_pam_users is an list, not an dict.